hazelcast/src/main/java/com/hazelcast/internal/networking/nio/iobalancer/IOBalancer.java

```diff
@@ -100,12 +100,12 @@ LoadTracker getOutLoadTracker() {
         return outLoadTracker;
     }
 
-    public void connectionAdded(MigratableHandler readHandler, MigratableHandler writeHandler) {
```

Comment:
Since we don't rely on connections any longer, but focus on channels; best to rename this to something more channel oriented.